valid-palindrome/gwbaik9717.js

```diff
@@ -0,0 +1,17 @@
+// Time complexity: O(n)
+// Space complexity: O(n)
+
+/**
+ * @param {string} s
+ * @return {boolean}
+ */
+var isPalindrome = function (s) {
+  const normalize = (s) => {
+    return s.toLowerCase().replace(/[^a-z0-9]/g, "");
+  };
+
+  const normalized = normalize(s);
```

Comment:
한 번밖에 안 쓰이는 함수인데 따로 변수에 할당할 필요가 있을까 하는 생각이 들었습니다 :)

```suggestion
  const normalized = s.toLowerCase().replace(/[^a-z0-9]/g, "");
```

Reply:
좋은 의견 주셔서 감사합니다! 말씀해주신 것 처럼 굳이 함수를 하나 더 만든 느낌이라서 해당 부분 반영하도록 하겠습니다 :)